java/ql/lib/change-notes/2024-03-12-request-sanitizers.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* Added sanitizers for relative URLs, `List.contains()`, and checking the host on an URI to the `java/ssrf` query.
```

Comment:
`java/unvalidated-url-redirection` shares these sanitizers.

```suggestion
* Added sanitizers for relative URLs, `List.contains()`, and checking the host on an URI to the `java/ssrf` and `java/unvalidated-url-redirection` queries.
```

Also, I think this should go below `src` instead of `lib`.